### ISSUE TITLE
Mask R-CNN example does not terminate on pytorch 1.2 and tv 0.4

### ISSUE THREAD

**Issue by shapovalov** [pytorch/vision#1328]
For my setup (pytorch 1.2 and tv 0.4), the default example for Mask R-CNN prediction does not terminate:
```
import torch, torchvision
model = torchvision.models.detection.maskrcnn_resnet50_fpn(pretrained=True)
model.eval()
x = [torch.rand(3, 300, 400), torch.rand(3, 500, 400)]
predictions = model(x)
```
Does not matter pretrained or not, CPU or GPU.
I have traced the problem to this function call: https://github.com/pytorch/vision/blob/master/torchvision/ops/poolers.py#L134
Any thoughts why it can be the case?

**Comment by fmassa** [comment]
This is weird.

I've just tried running the aforementioned script with PyTorch nightly (from today) and torchvision master and it run fine.

I don't currently have any ideas on why this is hanging. Can you try this snippet on colab and see if it works?

**Comment by fmassa** [comment]
Ok, thanks for letting me know! So it was probably a bad interaction somewhere.
I'm closing this issue then, as it seems it's fixed with an update.
